{
    "name": "@perfective/error",
    "version": "0.2.4",
    "description": "Functions for the Error and other types to handle exceptions",
    "keywords": [
        "error",
        "error chaining",
        "error handling",
        "exception",
        "exception chaining",
        "exception handling",
        "panic",
        "stack trace",
        "throwable",
        "es6",
        "es2015"
    ],
    "author": "Andrey Mikheychik <a.mikheychik@gmail.com>",
    "homepage": "https://github.com/perfective/js",
    "repository": {
        "type": "git",
        "url": "https://github.com/perfective/js.git",
        "directory": "packages/error"
    },
    "license": "MIT",
    "main": "dist/cjs/index.js",
    "module": "dist/mjs/index.js",
    "types": "dist/mjs/index.d.ts",
    "exports": {
        "import": "./dist/mjs/index.js",
        "require": "./dist/cjs/index.js"
    },
    "files": [
        "dist/cjs",
        "dist/mjs"
    ],
    "sideEffects": false,
    "scripts": {
        "build": "tsc --project ./tsconfig.build.cjs.json && tsc --project ./tsconfig.build.mjs.json",
        "build:clean": "npm run clean && npm run build",
        "clean": "rm -rf dist && rm -f *.tsbuildinfo",
        "lint": "npm run lint:eslint",
        "lint:eslint": "eslint --fix --ext .ts ./src",
        "test": "jest",
        "test:clean": "jest --clearCache",
        "test:verbose": "jest --verbose"
    }
}
